tony-core/src/main/java/com/linkedin/tony/TonyApplicationMaster.java

```diff
@@ -346,6 +390,29 @@ public static void main(String[] args) {
     }
   }
 
+  private static Metadata createMetadataObj(long started, long completed, boolean status) {
+    String jobStatus = status ? "SUCCEEDED" : "FAILED";
+    String url = "http://" + yarnConf.get("yarn.resourcemanager.webapp.address") + "/cluster/app/" + appIdString;
```

Comment:
You can use `YarnConfiguration.RM_WEBAPP_ADDRESS` instead of hardcoding `"yarn.resourcemanager.webapp.address"`.

Reply:
Noted. Fixed: https://github.com/linkedin/TonY/pull/79/commits/b67ee52d3644b2c199f8257c496395295f92b1ab